api/src/main/java/org/openmrs/module/reporting/report/renderer/IndicatorReportRenderer.java

```diff
@@ -31,8 +31,11 @@
  */
 //@Handler
 @Localized("reporting.IndicatorReportRenderer")
+@Deprecated
 public class IndicatorReportRenderer extends ReportDesignRenderer {
 
+
```

Comment:
@SsebaddukaG there are several unwanted empty lines added in your pull request. Can you please remove them ?

Reply:
yeah sure